Todo-script.js

```diff
@@ -1,47 +1,44 @@
-// Dodavanje "close" dugmeta za svaku stavku liste
-const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
-var variable = $("#userInput");
-
-// Ciljanje <ul> preko klase "todo-list"
-var todoList = $(".todo-list");
-
-$("li").each(function () {
-  $(this).append(SPAN_CLOSE);
-});
-
-// Klik na "close" dugme skriva trenutnu stavku
-todoList.on("click", ".close", function () {
-  $(this).parent().css("display", "none");
-});
-
-// Klik na stavku označava je kao "checked"
-todoList.on("click", "li", function () {
-  $(this).toggleClass("checked");
-});
-
-// Dodavanje nove stavke na listu
-$(".add-btn").click(function () {
-  var inputValue = variable.val();
-  if (!inputValue) {
-    showError("You must enter something on To-do list.");
-  } else {
-    var listItem = $("<li>").text(inputValue).append(SPAN_CLOSE);
-    todoList.append(listItem);
-    variable.val("");
+$(document).ready(() => {
+  const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
+  const POPOUP_TIME = 1500;
+  const POPUP_ERROR = "<div class='error-popup'>";
+  const ERROR_MESSAGE = "You must enter something on To-do list";
+  const USER_INPUT = $("#userInput");
```

Comment:
This one shouldnt be a constant, because this is a jquery constant which will search for the #userInput element only once it is defined.